pcsx2/gui/FrameForGS.cpp

```diff
@@ -557,6 +557,7 @@ void GSFrame::OnUpdateTitle( wxTimerEvent& evt )
 #endif
 
 	double fps = wxGetApp().FpsManager.GetFramerate();
+	float per = gsRegionMode == Region_NTSC ? (fps * 100) / EmuConfig.GS.FramerateNTSC.GetWhole() : (fps * 100) / EmuConfig.GS.FrameratePAL.GetWhole(); 
```

Comment:
@willkuer Theoretically, the rate could have higher accuracy than 1% or 2%, for instance if PCSX2 is fast enough for exactly 59.5 frames/sec where it should do 59.94 (possibly by doing one second with 59 frames, the next one with 60 frames, then 59 etc).

The rate updates currently twice a second but I think it could be changed.

Practically, however, I'm not sure you're wrong. Displaying integer percentage is certainly cleaner that with decimal point, and possibly doesn't throw away any practically useful information. We could start with integer and if people want more, reconsider it later.